Sources/AppcuesKit/Push/PushAutoConfig.swift

```diff
@@ -66,12 +66,20 @@ internal enum PushAutoConfig {
         let didHandleResponse = pushMonitors.contains { weakPushMonitor in
             guard let pushMonitor = weakPushMonitor.value else { return false }
 
+            // `completionHandler` is executed iff this returns true
```

Comment:
typo in comment (iff?)

Reply:
Actually intentional! _iff_ ie "[if and only if](https://en.wikipedia.org/wiki/If_and_only_if)"

Reply:
should it be `iff this returns false` ? Or am I misreading that?

Reply:
The closure block passed into `pushMonitor.didReceiveNotification` is called only when it returns true. Otherwise it's uncalled. Because this line is the return value in the `didHandleResponse = pushMonitors.contains` block, `didHandleResponse` then indicates if the completion handler has been called.

I definitely need to make this comment clearer, so I'll do that in a future PR 😅

Reply:
ah ok I follow now, thanks!